cloud/terraform/terraform_configurator.py

```diff
@@ -53,7 +53,7 @@ def get_config_builder(self):
         if cloud_name == 'aws':
             return AWSConfigBuilder(self.resources_dict, self.ssh_key_path)
         elif cloud_name == 'azure':
-            return AzureConfigBuilder(self.resources_dict, self.ssh_key_path)
+            return AzureConfigBuilderV2(self.resources_dict, self.ssh_key_path)
```

Comment:
By replacing this V1 version by the V2, we are braking current functionality which is e.g. running tests on images that come from the marketplace (see `cloud/sample/resources_azure_marketplace.json` and `cloud/sample/resources_azure.json`). so, we either implement a factory for this Azure config builders, or directly check if the `vhd_uri` element is present in the json, to chose to use the V2 one. Any other options/ideas?